### DISCUSSION TITLE
Vercel CLI 32.3.1 fails to recognise some node_module dependencies

### DISCUSSION THREAD

**Discussion by martin-dimi** [vercel/community#4231] [Help] [answered]
### Summary

Since our builds started using `Vercel CLI 32.3.1` we've been hitting run time issues with 3rd party dependencies:
```
Error: Cannot find module '/var/task/node_modules/playwright-core/index.mjs' imported from /var/task/bowser/api/_common.js
Did you mean to import playwright-core/index.js?
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:329:11)
    at moduleResolve (node:internal/modules/esm/resolve:992:10)
    at moduleResolveWithNodePath (node:internal/modules/esm/resolve:936:12)
    at defaultResolve (node:internal/modules/esm/resolve:1178:79)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
RequestId: 4405f3c7-cfac-41bb-a30b-8d36bd115b9d Error: Runtime exited with error: exit status 1
Runtime.ExitError
```

Nothing has changed in our code, all dependency versions are also locked so no updates there either. 
We've tried building a deployment from a week ago (that we know 100% used to work) and it still gives us the above runtime  error. 

I suspect it's a bug with Vercel CLI 32.3.1. Interestingly, running it locally works, it only breaks when deployed to vercel. 

### Steps to Reproduce

Run a serverless function that uses 
```
    "playwright-core": "1.31.1"
```
And build it using `Vercel CLI 32.3.1` on vercel.

**Comment by TooTallNate** [comment]
Could possibly have been caused by [this update to `@vercel/nft`](https://github.com/vercel/vercel/pull/10540).

**Reply by TooTallNate** [reply]
Can you try creating a deployment with environment variable with name `VERCEL_CLI_VERSION` set to value `https://vercel-git-revert-10540.vercel.sh/tarballs/vercel.tgz` and let us know if that fixes things for you?

**Reply by TooTallNate** [reply]
If that doesn't fix things, can you share a minimal reproduction so that we can take a closer look?

**Reply by martin-dimi** [reply]
Yes, can confirm setting the version to `32.3.1-ce83250` fixed the issue. 
How reliable is that version? This is blocking our prod deployments. Alternatively, can you link me 32.2.5 tarball which we can use till this gets resolved?

**Reply by TooTallNate** [reply]
You can just set `VERCEL_CLI_VERSION=vercel@32.2.5` to pin to a specific npm version. I'll report back here when we get that issue resolved.

**Reply by martin-dimi** [reply]
Nice! Thanks for unblocking us

**Comment by Ethan-Arrowood** [comment, marked as answer]
Hi @martin-dimi , 

Wanted to let you know that this problem has been resolved and you can now use the latest version of the CLI again. Thanks for the report!
